Hackney.Core/Hackney.Core.DynamoDb/Hackney.Core.DynamoDb.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-	 <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.0.21" />
+	 <PackageReference Include="AWSSDK.DynamoDBv2" Version=" 3.7.303.2" />
```

Comment:
Gonna assume the space at the start doesnt break it

Reply:
Even if it doesn't, would be nice to have the space removed :thinking:

Reply:
Fixed in https://github.com/LBHackney-IT/lbh-core/pull/62/commits/e6af791eb8e98763499cd609245eb648916ec97d